internal/pkg/db/memory/data.go

```diff
@@ -16,14 +16,15 @@ package memory
 import (
 	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
 	"github.com/edgexfoundry/edgex-go/pkg/models"
+	"github.com/google/uuid"
```

Comment:
That's another uuid library. Are we going to use `github.com/satori/go.uuid`, `gopkg.in/mgo.v2/bson` or `github.com/google/uuid`?

Reply:
The Google library is the preferred one. There's some existing code that uses the Satori library, but it doesn't seem to be as actively maintained as the Google one.